bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -0,0 +1,218 @@
+/*
+ * TesterBlobGranuleCorrectnessWorkload.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "TesterApiWorkload.h"
+#include "TesterUtil.h"
+#include <memory>
+#include <fmt/format.h>
+
+namespace FdbApiTester {
+
+class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
+public:
+	ApiBlobGranuleCorrectnessWorkload(const WorkloadConfig& config) : ApiWorkload(config) {
+		// sometimes don't do range clears
+		if (Random::get().randomInt(0, 1) == 0) {
+			excludedOpTypes.push_back(OP_CLEAR_RANGE);
+		}
+	}
+
+private:
+	enum OpType { OP_INSERT, OP_CLEAR, OP_CLEAR_RANGE, OP_READ, OP_GET_RANGES, OP_LAST = OP_GET_RANGES };
+	std::vector<OpType> excludedOpTypes;
+
+	void randomInsertOp(TTaskFct cont) {
```

Comment:
If randomInsertOp, randomClearOp, randomClearRangeOp are pure-copy-paste from ApiCorrrectnessWorkload maybe we should move them into a common base class and reuse

Reply:
They are, that sounds good

Reply:
I could move them into testerApiWorkload? Since it already has populateData and other functions that take a TTaskFct as an argument?

Reply:
Yes, we can move them into ApiWorkload class for now